security/pkg/pki/util/generate_cert.go

```diff
@@ -135,6 +135,11 @@ func LoadSignerCredsFromFiles(signerCertFile string, signerPrivFile string) (*x5
 	return cert, key, nil
 }
 
+// GenSubjectName returns the formatted subject name for the certificate (SPIFFEE format for now).
+func GenSubjectName(ns, serviceAccount string) string {
```

Comment:
GenSanUriName? As this is not the subject field in the certs.
Then maybe we can put this function in san.go:
https://github.com/istio/istio/blob/master/security/pkg/pki/util/san.go